testvectors_v1/mlkem_1024_decaps_seed_test.json

```diff
@@ -0,0 +1,1553 @@
+{
+  "algorithm": "ML-KEM",
+  "schema": "mlkem_decaps_seed_test_schema.json",
+  "numberOfTests": 192,
+  "testGroups": [
+    {
+      "type": "MLKEMDecaps",
+      "source": {
+        "name": "FIPS 203",
+        "version": "1.0"
+      },
+      "parameterSet": "ML-KEM-1024",
+      "tests": [
+        {
+          "tcId": 1,
+          "comment": "Test vector based on official round 3 seeds 0, seed: \"061550234d158c5ec95595fe04ef7a25767f2e24cc2bc479d09d86dc9abcfde7056a8c266f9ef97ed08541dbd2e1ffa1\"",
+          "seed": "7c9935a0b07694aa0c6d10e4db6b1add2fd81a25ccb148032dcd739936737f2d8626ed79d451140800e03b59b956f8210e556067407d13dc90fa9e8b872bfb8f",
```

Comment:
The original vectors from https://github.com/C2SP/wycheproof/pull/194 had a "seed" value in the comments, and then a "private_key" with the `d || z` seed.

In this rework to JSON you've renamed `private_key` to `seed`, which I think is the right choice, but now makes the `seed` in the comment stand out as a point of confusion.

I wonder if we should amend the comment to say "test generation seed" (assuming this is an accurate description) instead of "seed" to try and disambiguate from the decapsulation key seed. WDYT?